src/attributes.js

```diff
@@ -21,6 +21,23 @@ import {
   has
 } from './util';
 
+/**
+ * Reference map for SVG namespaced attributes
+ */
+const xlinkNS = 'http://www.w3.org/1999/xlink';
+const xmlNS = 'http://www.w3.org/XML/1998/namespace';
+const attributeNSMap = {
```

Comment:
We should make this prototype-less. There are some weird methods like `watch` on a default object.